frundenbot/main.py

```diff
@@ -58,124 +99,110 @@ def __log_incomming_messages(bot, update):
             target_chat += ' (%s)' % chat.username
         LOGGER.info('In:  %s: %s' % (target_chat, update.message.text))
 
-CACHE_REFRESH_TIME = Summary('frunde_cache_refresh_seconds', 'Time spent refreshing cache')
-FRUNDE_OPEN = Gauge('frunde_status', '1 if Frunde is open,-1 on error, 0 otherwise')
-@CACHE_REFRESH_TIME.time()
-def refresh_cache(context: CallbackContext):
-    global cache
-    try:
-        LOGGER.debug('Refresh cache')
-        r = requests.get('https://watchyour.freitagsrunde.org')
-        r.raise_for_status()
-        if 'Wir sind fuer dich da!' in r.text:
-            FRUNDE_OPEN.set(1)
-            cache = emojize(':white_check_mark: Die Freitagsrunde ist offen!', use_aliases=True)
+
+    START_TIME = Summary('frunde_start_seconds', 'Time spent executing /start handler')
+    @START_TIME.time()
+    def _callback_start(self, update: Update, context: CallbackContext):
+        context.bot.sendMessage(chat_id=update.message.chat_id, text='Egal was du sagst, ich sag nur, ob die Freitagsrunde offen hat! Du kannst mich direkt anschreiben oder inline per @FrundenBot in anderen Chats benutzen.')
+
+
+    @command(name='commands', description='List commands supported by this bot.')
```

Comment:
Fyi: In my latest bot I changed the name of this to `['help', 'h']`, since it is more in line with actual command line tools.

Reply:
Makes sense :)